cmd/launcher/uninstall.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/peterbourgon/ff/v3"
+)
+
+var rootDirRegexp = regexp.MustCompile(`^root_directory \/var\/([-a-zA-Z0-9]*)\/.*\.kolide\.com`)
+
+func runUninstall(args []string) error {
+	var (
+		flagset  = flag.NewFlagSet("kolide uninstaller", flag.ExitOnError)
+		flconfig = flagset.String(
+			"config",
+			"",
+			"launcher flags configuration file",
+		)
+	)
+
+	if err := ff.Parse(flagset, args, ff.WithEnvVarNoPrefix()); err != nil {
+		return fmt.Errorf("parsing flags: %w", err)
+	}
+
+	// Attempt to get the identifier from the config file if one was provided
+	identifier, err := getIdentifierFromConfigFile(*flconfig)
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	return removeLauncher(ctx, identifier)
+}
+
+func getIdentifierFromConfigFile(configFilePath string) (string, error) {
```

Comment:
Hrm... This feels a a little weird. 

I'd probably use `ff`'s `WithIgnoreUndefined` option. And parse it the same way we normally parse the config.

Reply:
Oh wow, I wasn't aware of that feature in ff - that's way better!